sherpa-onnx/csrc/math.h

```diff
@@ -103,5 +103,18 @@ std::vector<int32_t> TopkIndex(const T *vec, int32_t size, int32_t topk) {
   return index;
 }
 
+template <class T>
```

Comment:
Please follow
https://github.com/k2-fsa/sherpa-onnx/blob/80060c276d2d5d75c9b2f24a132d7694ed1e4942/sherpa-onnx/csrc/offline-ctc-greedy-search-decoder.cc#L35-L39

to use `std::max_element` to replace `ArgMax()`.

You don't need to reimplement the wheel.

Reply:
> `p_log_probs` is `const float *`, why do we use `static_cast` again?

yes, you are right. please remove the cast.